config/routes.rb

```diff
@@ -6,13 +6,23 @@
   root 'task#index'
   # get '/' => 'welcome#index'
   # lines 6 and 7 are equivalent
-  get 'task/new' => 'task#new'
+  get '/task/new' => 'task#new'
   post '/tasks' => 'task#create'
 
-  delete '/task/:id' => 'task#delete'
+  delete '/delete/:id' => 'task#destroy'
+
+  patch '/task/:id' => 'task#mark_complete'
+
+  get '/task/:id/edit' => 'task#edit'
+  patch '/task/:id/edit' => 'task#update'
 
   get '/task/:id' => 'task#show'
 
+
+  get '/people' => 'people#index'
+  get '/people/:id' => 'people#show'
+  get '/people/:id/tasks' => 'people#all_tasks'
+
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
```

Comment:
Your route are set up very well - when/if you updated you tasks controller you'd also want to update your task routes to be plural